src/test/integration/User.t.sol

```diff
@@ -123,6 +123,16 @@ contract User is Test {
         delegationManager.delegateTo(address(operator), emptySig, bytes32(0));
     }
 
+    /// @dev Undelegate from operator
+    function undelegate() public createSnapshot virtual returns(bytes32){
+        return delegationManager.undelegate(address(this));
+    }
+
+    /// @dev Force undelegate staker
+    function undelegate(User staker) public createSnapshot virtual returns(bytes32){
+        return delegationManager.undelegate(address(staker));
+    }
```

Comment:
Could we rename this `forceUndelegate`?

It matches the dmgr interface less, but I think it's clearer at the test level